src/index.js

```diff
@@ -0,0 +1,193 @@
+    </div>
+    <!-- Food Name -->
+    <div class="popup-name-container"><h2>${foodObject.name}</h2></div>
+    <!-- Food Details -->
+    <div class="popup-details-container">
+    <div class="popup-details-upper"><span>Count: ${foodObject.count}</span><span>ID: ${foodObject.id}</span></div>
+    <div class="popup-details-lower"><span>Likes: ${foodObject.likes}</span><span>Delicious: Yes</span></div>
+    </div>
+    <!-- Comments and comment Details -->
+    <div class="popup-comments-container">
+    <h2>Comments (${foodObject.comments.length})</h2>
+    <div class="popup-comments">${commentsObjectRender(foodObject.comments)}</div>
+    <!-- Add a Comment -->
+    <div class="add-comment">
+        <h2>Add a Comment</h2>
+        <form class="add-comment-form">
+        <input type="text" name="name" id="name" placeholder="Your Name" required/>
+        <textArea type="text" name="comment" id="comment" placeholder="Your Insights" cols=50 maxLength=500 required></textArea>
+        <button type="submit" class="add-comment-button">Comment</button>
+        </form>
+    </div>
+    </div>
+    `;
+
+    popupContainerBackground.appendChild(popupContainer);
+    document.body.appendChild(popupContainerBackground);
+    document.querySelector('.close-button').addEventListener('click', () => {
+        popupContainerBackground.remove();
+    });
+    document.querySelector('.add-comment-form').addEventListener('submit', async (e) => {
+        e.preventDefault();
+        const date = new Date();
+        const name = document.querySelector('#name').value;
+        const comment = document.querySelector('#comment').value;
+        const resp = await addComment(date, foodObject.id, name, comment);
+        if (resp === 'Created') {
+            const commentElement = document.createElement('div');
+            commentElement.classList.add('individual-comment');
+            commentElement.innerText = `${date.getMonth() + 1}/${date.getDate()}/${date.getFullYear()} ${name}: ${comment}`;
+            document.querySelector('.popup-comments').appendChild(commentElement);
+            foodObject.comments.push({
+                comment: `${date.getMonth() + 1}/${date.getDate()}/${date.getFullYear()} ${name}: ${comment}`,
+                creation_date: `${date.getFullYear()}-${date.getMonth() + 1}-${date.getDate()}`,
+                username: name,
+            });
+            document.querySelector('.popup-comments-container h2').innerText = `Comments (${foodObject.comments.length})`;
+        };
+        document.querySelector('#name').value = '';
+        document.querySelector('#comment').value = '';
+    });
+};
+
+document.addEventListener('DOMContentLoaded', async () => {
+  appRender();
+
+  const foodByChina = await foodByChinaList();
+
+  console.log(foodByChina);
+
+  // Show items count on homepage Header
+    const foodCount = document.querySelector('nav > ul > li > a');
+    foodCount.innerHTML = `Chinese(${foodByChina.length})`;
```

Comment:
Here

Reply:
already tests are there and successfully passing!